Gruntfile.js

```diff
@@ -1,77 +1,128 @@
 module.exports = function(grunt) {
+    var headless = require('mocha-headless-chrome'),
+        _ = require('lodash'),
```

Comment:
Does this mean we now depend on both underscore and lodash?

Reply:
lodash was already installed as a dependency from one of the other packages, so I just ended up using it and requiring it in `package.json`. Arguably, we should move to `lodash` in favor of ditching `underscore`, but right now `npm`'s `lodash` and `bower`'s `underscore` are in completely different places / different use cases